src/Search.php

```diff
@@ -2329,6 +2329,12 @@ class='normalcriteria fold-search list-group-item p-2 border-0'
          }
 
          $(document).on("click", ".remove-search-criteria", function() {
+            // force removal of tooltip
+            var tooltip = bootstrap.Tooltip.getInstance($(this)[0]);
+            if (!!tooltip) {
```

Comment:
Returned value will be null, if no instance is related to passed element.
```suggestion
            if (tooltip !== null) {
```